enzy_htp/__init__.py

```diff
@@ -48,4 +47,11 @@
 end users and instead exists for developers to access external software. See enzy_htp/_interface/interface.py
 for full class defintion."""
 
-from .analysis import ( electric_field )
+config = interface.config() 
```

Comment:
wont this overwrite what's in line 38.

Reply:
It will! It's a no-op but still dumb. I have removed it.